setup_scripts/install_arduino.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+###################
+# Install Arduino #
+###################
+
+echo "================================================================"
+echo "Installing Arduino"
+echo "================================================================"
+ARDUINO_INSTALL_DIR=/usr/share
+ARDUINO_DIR_NAME=arduino
+ARDUINO_ARCHIVE_NAME=$ARDUINO_DIR_NAME.tar.xz
+
+echo "Downloading Arduino"
+sudo rm -rf /tmp/$ARDUINO_DIR_NAME
+sudo rm -rf /tmp/$ARDUINO_ARCHIVE_NAME
+wget http://downloads.arduino.cc/arduino-1.8.5-linux64.tar.xz -O /tmp/$ARDUINO_ARCHIVE_NAME
+
+echo "Extracting Arduino"
+mkdir /tmp/$ARDUINO_DIR_NAME
+tar xpvf /tmp/$ARDUINO_ARCHIVE_NAME -C /tmp/$ARDUINO_DIR_NAME --strip-components 1 > /dev/null
+sudo rm -rf $ARDUINO_INSTALL_DIR/$ARDUINO_DIR_NAME
+sudo mv /tmp/$ARDUINO_DIR_NAME $ARDUINO_INSTALL_DIR
+
+echo "Running the Arduino install script"
+$ARDUINO_INSTALL_DIR/$ARDUINO_DIR_NAME/install.sh
+
+echo "Linking Arduino"
+sudo rm /usr/local/bin/arduino
+sudo ln -s -f $ARDUINO_INSTALL_DIR/$ARDUINO_DIR_NAME/arduino /usr/local/bin/arduino
+
+echo "Applying Snowbots modifications to core Arduino files"
+
+## Add the `flushRX` function. For context, see issue #176
+# Add the implementation to the `.cpp` file
+# Remove the last line (the final `#endif`) so we're inside the guards
+sed -i '$ d' $ARDUINO_INSTALL_DIR/$ARDUINO_DIR_NAME/hardware/arduino/avr/cores/arduino/HardwareSerial.cpp
```

Comment:
Function declaration in the header file?

Reply:
Could we actually have someone look at if that is necessary, maybe it was just me being religious about it (flushRX). If the we don't actually drop messages, then we should remove this probably, unless someone has also noticed this fixing the problem. Who worked on this last year?

Reply:
Good catch, I just went through all the firmware code and it looks like we're no longer using the `flushRX` function, so I've removed this section of the setup script.